### PR TITLE
fix rdb datasource instance

### PR DESCRIPTION
**Pull request by Monitob**
- fix(rdb-instance):  remake cassettes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@ test.tf
 *.test
 *.iml
 .vscode
+.env
```
Comment (remyleone) [inline .gitignore:29]: Why is this change required? ignore of development environment can be done in a gitignore_global

File: scaleway/data_source_rdb_instance_test.go [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
 	"testing"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestAccScalewayDataSourceRdbInstance_Basic(t *testing.T) {
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
+	// instance name should be unique
```
Comment (remyleone) [inline scaleway/data_source_rdb_instance_test.go:14]: Why? In particular, why should it be unique and random?

Comment (Monitob) [reply scaleway/data_source_rdb_instance_test.go:14]: Well, I'm not available to reproduce this error. I'll rollback to not random name

**Comment by remyleone** [comment]
Test seems to panic after 1 hour https://github.com/scaleway/terraform-provider-scaleway/runs/4366706884?check_suite_focus=true#step:4:547

**Comment by remyleone** [comment]
I also don't understand why rdb intervene in the IoT test suite. What is the isolation level between the different tests https://github.com/scaleway/terraform-provider-scaleway/runs/4366706617?check_suite_focus=true#step:4:264 ?

**Comment by Monitob** [comment]
@remyleone rdb was used on the test steps on IOT [routes](https://github.com/scaleway/terraform-provider-scaleway/blob/26d08a3a716696f530e57151e12eb797e32fc6d8/scaleway/resource_iot_routes_test.go#L54)

**Pull request closed without merging**